stl/inc/regex

```diff
@@ -935,7 +935,7 @@ basic_ostream<_Elem, _Traits>& operator<<(basic_ostream<_Elem, _Traits>& _Ostr,
 }
 
 // FORWARD DECLARATIONS
-template <class _BidIt, class _Allocator = allocator<sub_match<_BidIt>>>
+template <class _BidIt, class _Alloc = allocator<sub_match<_BidIt>>>

```

Comment:
Note: this change can be reverted if the _Allocator concept is removed.

Reply:
`concept _Allocator` is still present, and this change is desirable on its own merits because the declaration now matches the definition below:
https://github.com/microsoft/STL/blob/ac4fde764ae716512bdd67e0e9577bd53947bc3d/stl/inc/regex#L951-L952